app/views/content/is-teaching-right-for-me/behaviour-management.md

```diff
@@ -0,0 +1,95 @@
+date: "2024-01-09"
+backlink: "../../"
+navigation: 5.35
+navigation_title: How can I manage behaviour in the classroom?
+navigation_description: Teacher training adviser Jane Wilkinson answers common concerns around behaviour management.
+promo_content:
+    - content/train-to-be-a-teacher/promos/mailing-list-promo-salaries
+related_content:
+    How to become a teacher : "/steps-to-become-a-teacher"
+    What to expect in teacher training : "/train-to-be-a-teacher/initial-teacher-training" 
+    Early career teachers (ECTs) : "/support-for-early-career-teachers"
+
+keywords:
+  - Behaviour management
+  - Managing behaviour
+  - Disruptive behaviour
+  - Challenging behaviour
+  - Disruptive pupils
+  - Difficult pupils
+  - Positive behaviour
+  - Classroom teaching
+  - NQT
+  - National professional qualification
+  - Support
+  - Training
+  - Teacher training
+  - ECF
+  - Teaching
+  - How to become a teacher
+  - Teacher training courses
+  - Behaviour
+  - Management
+  - classroom concerns
+---
+
+Teacher training adviser Jane Wilkinson has years of teaching experience. She answers some common concerns that trainee teachers have around how to manage behaviour in the classroom.
+
+## How can I create a positive learning environment?
+In my experience of teaching hundreds of thousands of children, if they’re engrossed in their learning and know what it is they need to be doing, they'll be much less disruptive. The biggest challenge isn't usually big things it’s more the chatting and low-level disruption.  
+
+Make sure you’re very clear. If children understand your expectations of them in terms of workload, behaviour and outcome and you scaffold that journey then behaviour becomes much less of a problem. Start off by modelling what you would want in your classroom. Do you want the children lining up at the start of the lesson? Or do you want them coming straight in and starting with the work?
+
+Have routines and rituals or standard operational procedures so children know when they go to your class, this is what they do. So they understand what's required from them and know they'll be rewarded if they get that behaviour right.
+
+## How do I manage more disruptive behaviour?
+Managing behaviour is a learning curve, and exactly what a teacher training course is designed for. You'll be offered practical guidance and support so don’t worry if you don’t get it right every time.
+
+It’s all about the relationships you build with pupils. If they know they’ll get away with disruptive learning, they'll disrupt learning.
+You have to set and communicate firm but fair boundaries so they know they're safe and won't be ridiculed or embarrassed and their learning comes first. They’ll push against these boundaries because that’s what children do. But if they find they meet with a brick wall they won't push any further because they know what's going to happen in any lesson they turn up to. 
```

Comment:
```suggestion
You have to set and communicate firm but fair boundaries so they know they're safe and won't be ridiculed or embarrassed and their learning comes first. They’ll push against these boundaries because that’s what children do. But if they are met with a brick wall they won't push any further because they know what's going to happen in any lesson they turn up to. 
```